dev/archery/archery/crossbow/core.py

```diff
@@ -299,9 +299,9 @@ def push(self, refs=None, github_token=None):
         try:
             self.origin.push(refs + self._updated_refs, callbacks=callbacks)
         except pygit2.GitError:
-            raise RuntimeError('Failed to push updated references, '
-                               'potentially because of credential issues: {}'
-                               .format(self._updated_refs))
+            raise RuntimeError(f"Failed to push updated references, "
```

Comment:
`f` is not needed here

Reply:
Resolved in [2c7c552](https://github.com/apache/arrow/pull/45629/commits/2c7c552cf2119490867016bdce97578af1f1481b).